src/templates/_common/includes/mfsa.jsx

```diff
@@ -7,7 +7,8 @@ const MFSA = () => {
     return (
         <div style={style} className='container mfsa_message'>
             <div className='notice-msg center-text gr-parent gr-child'>
-                <p>{it.L('In the EU, financial binary options are only available to professional investors.')}</p>
+                <p>{it.L('In the EU, financial binary options are only available to professional investors.')} <a id='professional-cta' className='invisible' href={it.url_for('user/settings/professional')}>{it.L('Apply now as a professional investor')}</a>
```

Comment:
would be better to use `&nbsp;` instead of the white space before `<a`

Reply:
Do HTML entities get rendered in JSX components?

Reply:
can you look through the code and find the answer to that? 🙂